reflex/vars/base.py

```diff
@@ -2537,3 +2556,38 @@ def _extract_var_data(value: Iterable) -> list[VarData | None]:
     "set_state": "_var_set_state",
     "deps": "_deps",
 }
+
+
+def eval_component(var_name: str) -> Tuple[str, VarData]:
```

Comment:
this should probably end up in the same module as `make_component`

maybe in `reflex/components/dynamic.py`?